core/runtime/wasm_result.hpp

```diff
@@ -14,15 +14,15 @@ namespace kagome::runtime {
    * bits are the address and next 32 bits are the size of the returned buffer.
    */
   struct WasmResult {
-    explicit constexpr WasmResult(WasmSpan v)
-        : address(getWasmAddr(v)), length(getWasmLen(v)) {}
+    explicit constexpr WasmResult(WasmSpan v) {
+      auto [ptr, len] = splitSpan(v);
+      address = ptr;
+      length = len;
+    }
```

Comment:
May be change it to be type-alias of WasmSpan? Or trivial descendant of WasmSpan.

Reply:
I was actually thinking of renaming it to WasmSpan, but preferred to keep it for a separate PR.

Reply:
Ok, good